Objects/obmalloc.c

```diff
@@ -31,8 +31,8 @@ static void _PyMem_SetupDebugHooksDomain(PyMemAllocatorDomain domain);
 
 #if defined(__has_feature)  /* Clang */
 #  if __has_feature(address_sanitizer) /* is ASAN enabled? */
-#    define _Py_NO_ADDRESS_SAFETY_ANALYSIS \
-        __attribute__((no_address_safety_analysis))
+#    define _Py_NO_SANITIZE_ADDRESS \
+        __attribute__((no_sanitize("address")))
```

Comment:
Shouldn't this be `no_sanitize_address`?

Reply:
The [page](https://clang.llvm.org/docs/AddressSanitizer.html#disabling-instrumentation-with-attribute-no-sanitize-address) issue refers suggests usage of this instead of other deprecated aliases

> One may use the attribute `__attribute__((no_sanitize("address")))` (which has deprecated synonyms no_sanitize_address and no_address_safety_analysis) to disable instrumentation of a particular function.